js/books.js

```diff
@@ -0,0 +1,52 @@
+export default class Bookshelf {
+    #books;
+
+    constructor() {
+      this.#books = {};
+    }
+
+    // Setters
+    SetBooks(books) {
+      this.#books = books;
+    }
+
+    // Getter
+    get Books() {
+      return this.#books;
+    }
+
+    // Methods
+    AddBook(title, author) {
+      const id = `${new Date().getTime()}`;
+      this.#books.push({
+        title,
+        author,
+        id,
+      });
+    }
+
+    DeleteBook(id) {
+      const localBooks = this.#books;
+
+      this.#books = localBooks.filter((books) => {
+        if (books.id !== id) {
+          return true;
+        }
+        return false;
+      });
+    }
+
+    SaveBookshelfLocal() {
+      localStorage.setItem('books', JSON.stringify(this.#books));
+    }
+
+    LoadBooksFromLocal() {
+      const savedBooks = JSON.parse(localStorage.getItem('books'));
+
+      if (Array.isArray(savedBooks)) {
+        this.#books = savedBooks;
+        return true;
+      }
+      return false;
+    }
```

Comment:
- [ ] Kindly make sure your page is able to save data in the Local Storage as required for the project that your application should keep all functionality. Right now once you reload your page, everything from the local storage disappears and nothing could not be displayed on your books list(as the local storage is empty)